pkg/tags/tag.go

```diff
@@ -172,15 +172,6 @@ func (t *Tag) Done(s State) bool {
 	return err == nil && n == total
 }
 
-// DoneSplit sets total count to SPLIT count and sets the associated swarm hash for this tag
```

Comment:
why is this removed? it is important when you  upload from stream of unknown size

Reply:
Total is incremented inside splitter when a chunk is created.  So no need of calling a separate DoneSplit outside.

Reply:
ok fine but that means that progress bars will not be able to say 'we. dont know. yet'

Reply:
hmm.. yes. and i am not okay with it. I will add DoneSplit